scene/main/viewport.h

```diff
@@ -186,6 +186,12 @@ class Viewport : public Node {
 	RID contact_2d_debug;
 	RID contact_3d_debug_multimesh;
 	RID contact_3d_debug_instance;
+	SpatialMaterial *contact_3d_debug_mat;
```

Comment:
This will likely fail compiling with `disable_3d=yes`, as `SpatialMaterial` is not registered in ClassDB then.

Reply:
I'm going to use _3D_DISABLED is it ok?

Reply:
Yeah that should do the trick.

Reply:
Done